client/src/__tests__/integration/news-slug.test.js

```diff
@@ -0,0 +1,28 @@
+/* global describe it expect */
+import faker from 'faker';
+import { kebabCase } from 'lodash';
+import shortId from 'shortid';
+
+import { createArticleSlug } from '../../../utils/news';
+import { getShortIdFromSlug } from '../../utils';
+
+describe('news-slug integration', () => {
+  it('returns the correct id from a generated slug', () => {
+    expect.assertions(100);
```

Comment:
Since this is the third use (by my count), maybe a shared named constant is in order. Easier to change if/when that needs to be done.

Reply:
I don't understand what you mean? `expect.assertions` is part of the [`jest` API](https://jestjs.io/docs/en/expect#expectassertionsnumber)

Reply:
the 100.

Reply:
This file is part of a test suite and not used in production, there is no-where else we use the number `100` in a test suite?

Any other instances of `100` in this PR relate to parameters to, and default arguments for the `fcc-source-news` plugin.

I still don't understand what you mean?

Reply:
Exactly what you wrote in the second paragraph.